### DISCUSSION TITLE
Classes estáticas - dúvida de uso

### DISCUSSION THREAD

**Discussion by potumati** [balta-io/forum#491] [Tira Dúvidas] [answered]
Eu tenho algumas classes/métodos estáticos, principalmente para tratamento de strings.

Por exemplo, tenho uma 
`public static string GetYoutubeId(string youtubeVideoUrl) { }`

O ideal é eu manter como estática ou ter que instanciar toda vez?

**Comment by andrebaltieri** [comment, marked as answer]
@potumati boa tarde, como vai?

Depende muito do cenário, eu tenho várias classes aqui para os mesmos fins e a maioria também é estática, inclusive com uso dos Extension Methods.

Se me falar um cenário específico posso ser mais assertivo na resposta.
